### DISCUSSION TITLE
چت پروژه

### DISCUSSION THREAD

**Discussion by hashemian1382** [advanced-programming-sut-2023/project-group-36#33] [General]
چت پروژه / گروه طور

**Comment by hashemian1382** [comment]
سلام محمد بیا اینجا

**Reply by mohammad6987** [reply]
اومدم

**Reply by mohammad6987** [reply]
دراپ بیلدینگ رو الان جابجاش میکنم

**Reply by hashemian1382** [reply]
خب برای اون دراپ بیلدینگ داخل ادیت مپ چی؟

**Reply by hashemian1382** [reply]
دستور سلکت ساختمان هم حواست باشه بهش
کلا اینطوریه که توی بازی هر حکومت یه سلکتد ساختمان داره که البته میتونه null باشه
برای خیلی کارا لازمه ساختمان انتخاب شده باشه
مثلا برای ساخت سرباز 
یا تعمیر ساختمان

**Reply by hashemian1382** [reply]
ریسلکت هم صرفا null میکنه سلکتد ساختمان رو

**Reply by hashemian1382** [reply]
دستور تعمیر هم میاد اون ساختمان رو هیت پوینتش رو فول میکنه اگه منابع کافی باشه البته و یه سری شرط دیگه

**Reply by mohammad6987** [reply]
> خب برای اون دراپ بیلدینگ داخل ادیت مپ چی؟

برای اون هم چندتا ساختمون پیشفرض میذارم

**Reply by mohammad6987** [reply]
> دستور تعمیر هم میاد اون ساختمان رو هیت پوینتش رو فول میکنه اگه منابع کافی باشه البته و یه سری شرط دیگه

تابع چک کردن منابع رو زدم،اونم درست میشه الان

**Comment by hashemian1382** [comment]
درباره ی دراپ بیلدینگ ادیت مپ منظورت چیه چندتا ساختمون پیشفرض میذارییی؟
نباید پیش فرض باشه ها
اولین نفری که داره بازی رو میسازه باید بتونه بذاره
قسمت قرار دادن سرباز ها داخل همون ادیت مپ رو ببین که من زدم
عین همون بزن
طرف یه دستور وارد میکنه
که فلان ساختمون رو در فلان موقعیت بساز
تو میای میسازیش
فقط چک میکنی نامش ولید باشه همین
مثل همون مدلی که من واسه دراپ یونیت زدم بزنش

**Reply by mohammad6987** [reply]
اوکی
یدونه مقر اجباریه
دیگه چی هارو بذارم؟

**Reply by hashemian1382** [reply]
نه نه مقر رو نباید بذاری محمد اونو من باید بذارم
مقر همون جاییه که توی دستور ست گاورنمنت میگه طرف

**Reply by mohammad6987** [reply]
من توی ذهنم فقط همون مقر بودش
اخه منطقی نیست اول بازی بدون منابع ساختمون های دیگه رو بذاری

**Reply by hashemian1382** [reply]
محمد داکو برو ببین
قسمت قرار دادن عناصر
میشه گذاشت
ربطی هم به منابع نداره
اینا قبل شروع بازی هستش
هر نوع ساختمونی هم شامل میشه
کلا اینطوریه که اول طرف ست گاورنمنت رو میزنه و موقعیت و رنگ یه حکومت رو مشخص میکنه، بعد میاد سرباز ها و ساختمان های مربوط به اون رو قرار میده تا اینکه حکومت بعدی ست بشه...

**Reply by mohammad6987** [reply]
اسم تایپ هایی که برای جنس زمین گذاشتی رو میگی؟

**Reply by mohammad6987** [reply]
حاجی اینو بگووو

**Reply by mohammad6987** [reply]
برای دراپ بیلدینگ معدن و مزرعه میخوام

**Comment by mohammad6987** [comment]
![image](https://github.com/advanced-programming-sut-2023/project-group-36/assets/119624288/2f69c2e8-8e46-4e61-b16b-c32ff23bd289)

**Reply by mohammad6987** [reply]
رجکس ها درست کار میکنن؟

**Reply by hashemian1382** [reply]
انشالله که درست کار میکنن
با چیزایی که من امروز دیدم حقیقتا احتمالش کمه :)

**Comment by hashemian1382** [comment]
محمد قرار دادن ساختمان توی ادیت مپ رو چیکار کردی؟

**Reply by mohammad6987** [reply]
دارم میزنمش الان
